src/api/app/components/request_decision_component.html.haml

```diff
@@ -1,33 +1,97 @@
+            .dropdown-menu{ aria: { labelledby: 'decision-buttons-group' } }
+              -# Simple Accept request submit
+              = submit_tag('Accept request', name: 'accepted', class: 'btn-link dropdown-item', data: confirmation,
+                            title: 'Accept the request, this will apply the changes on the target.')
+              -# Accept and make maintainer submit
+              - if accept_with_options_allowed? && show_add_submitter_as_maintainer_option?
+                = submit_tag("Accept and make #{@bs_request.creator} maintainer of #{make_maintainer_of}",
+                            name: 'accepted',
+                            class: 'btn-link dropdown-item', data: confirmation,
+                            title: 'Accept the request. this will apply the changes on the target.',
+                            id: 'accept-and-make-maintainer')
+          -# Accept and forward buttons
+          - if forward_allowed?
+            %span.dropdown.me-2
+              = button_tag('Accept and Forward', type: 'button', role: 'button', id: 'decision-forward-buttons-group',
+                            class: 'btn btn-secondary dropdown-toggle',
+                            data: { 'bs-toggle': 'dropdown'}, aria: { 'haspopup': 'true', 'expanded': 'false' })
+              .dropdown-menu{ aria: { labelledby: 'decision-forward-buttons-group' } }
+                - @action.forward.each_with_index do |forward, index|
+                  .form-check.m-2
+                    = check_box_tag("forward-#{index}", "#{forward[:project]}_#_#{forward[:package]}", true,
+                                    class: 'form-check-input forward-check-inputs')
+                    %label.form-check-label{ for: "forward-#{index}" }
+                      Forward submit request to
+                      #{helpers.project_or_package_link(project: forward[:project], package: forward[:package], short: true)}
+                %hr.dropdown-divider
+                -# Accept and forward submit
+                = submit_tag('Accept and forward request to the selected projects ', name: 'accepted', class: 'btn-link dropdown-item',
+                              data: confirmation, title: 'Accept the request. this will apply the changes on the target.',
+                              id: 'accept-and-forward')
+                -# Accept and make maintainer and forward submit
+                - if show_add_submitter_as_maintainer_option?
+                  = submit_tag("Accept and make #{@bs_request.creator} maintainer of #{make_maintainer_of} and forward request to the selected projects",
+                                name: 'accepted', class: 'btn-link dropdown-item', data: confirmation,
+                                title: 'Accept the request. this will apply the changes on the target.',
+                                id: 'accept-and-make-maintainer-and-forward')
+
```

Comment:
Do you need a second dropdown? The only visible difference between the two dropdowns are the checkboxes, which could just be integrated into the first dropdown instead